ui/pages/settings/networks-tab/networks-form/networks-form.js

```diff
@@ -968,6 +969,9 @@ const NetworksForm = ({
               MetaMetricsNetworkEventSource.CustomNetworkForm,
             token_symbol: ticker,
           },
+          sensitiveProperties: {
+            rpcUrl: sha256(rpcUrl),
```

Comment:
```suggestion
            rpcUrl: await sha256(rpcUrl),
```

`sha256` is async and needs to be awaited.  Otherwise it's getting added to the event as a promise:
<img width="165" alt="Screenshot 2024-07-30 at 10 28 30 AM" src="https://github.com/user-attachments/assets/e381e91f-cc27-44df-b156-0217c0c97d14">